etcdctl/main.go

```diff
@@ -27,7 +27,8 @@ const (
 	apiEnv = "ETCDCTL_API"
 )
 
-/**
+/*
+*
```

Comment:
Seems like the second asterisk `*` should be deleted? Looks like a mandatory rule of the new Go tools.